framework/include/task_manager/task_manager.h

```diff
@@ -102,6 +103,21 @@ enum tm_defined_broadcast_msg {
 	TM_BROADCAST_MSG_MAX = TM_BROADCAST_SYSTEM_MSG_MAX,
 #endif
 };
+/**
```

Comment:
Could you please leave a line's space after the previous structure definition?

Reply:
Done

Reply:
Done.